### ISSUE TITLE
[Screen reader - Logic Apps - Logic app designer]: Screen reader does not announce any information when focus lands on the "+" controls.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6106]
### Describe the issue 

User Experience:

 Visually impaired users who rely on screen readers for navigation will be impacted if screen reader does not announce any information of the "+" controls as they will not get the purpose of the controls and will be disoriented.

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic Apps' and select it. Select any already created logic app 
- Select 'Logic App Designer' in the left navigation pane. 
- Go to Templates and Select any template like 'HTTP Request Response' . 
- Navigate all the controls on the 'HTTP Request Response' pop up. Press 'Use this template' button. Press the Save Button. 
- Make some changes to the workflow and Press Discard button. Navigate all the controls on the dialog box that appears. 
- Navigate all the controls on the 'Logic App Designer' blade like top menu bar (Save, Discard, Run, Designer, Code View, Templates, Connectors, Hep) , Designer section below the menu ..etc 
- Verify whether screen reader announces any information when focus lands on the "+" controls. 

Actual Result:

 The screen reader does not announce any information when focus lands on the "+" controls.

 Observation:

 The screen reader remains silent when focus lands on the "+" controls.

Expected Result:

 The screen reader should announce the information when focus lands on the "+" controls. Screen reader should announce the information like "Insert a new step between Recurrence and Start virtual machine button".

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29399683

**Comment by hartra344** [comment]
Unable to repro

**Comment by hartra344** [comment]
repros in windows only
